test/mason/mason-test-exit/exitCodeTest.chpl

```diff
@@ -3,14 +3,13 @@ use MasonUtils;
 use Spawn;
 
 
-proc checkExitStatus(cmd: string) {
-  var splitCmd = cmd.split();
-  var p = spawn(splitCmd,stdout=PIPE);
+proc checkExitStatus(cmd) {
```

Comment:
Was there an importance behind this change, or is it just a stylistic refactor?

Reply:
Just stylistic to match a few other of the tests